content/about/notes/1.0/index.md

```diff
@@ -3,18 +3,85 @@ title: Istio 1.0
+relative to 0.8 as most of the effort for this release went into fixing bugs and improving performance.
 
 ## Networking
 
+- **SNI Routing using Virtual Services**. Newly introduced `TLS` sections in
+[`VirtualService`](/docs/reference/config/istio.networking.v1alpha3/#VirtualService) can be used to route TLS traffic
+based on SNI values. Service ports named as TLS/HTTPS can be used in conjunction with
+virtual service TLS routes. TLS/HTTPS ports without an accompanying virtual service will be treated as opaque TCP.
+
+- **Old (v1alpha1) networking APIs are no longer available**. Support for the old `v1alpha1` traffic management model
+has been removed.
+
+- **Istio Ingress Deprecated**. The old Istio ingress is deprecated and disabled by default. We encourage users to use [gateways](/docs/concepts/traffic-management/#gateways) instead.
+
+- **Support for streaming gRPC restored**. Istio 0.8 caused periodic termination of long running streaming gRPC connections. This has been fixed in 1.0.
+
+## Policy and Telemetry
+
+- **Updated Attributes**. The set of [attributes](/docs/reference/config/policy-and-telemetry/attribute-vocabulary/) used to describe the source and
+destination of traffic have been completely revamped in order to be more
+precise and comprehensive.
+
+- **Policy Check Cache**. Mixer now features a large level 2 cache for policy checks, complementing the level 1 cache
+present in the sidecar proxy. This further reduces the average latency of externally-enforced
+policy checks.
+
+- **Telemetry Buffering**. Mixer now buffers report calls before dispatching to adapters, which gives an opportunity for
+adapters to process telemetry data in bigger chunks, reducing overall computational overhead
+in Mixer and its adapters.
+
+- **Out of Process Adapters**. Mixer now includes initial support for out-of-process adapters. This will
+be the recommended approach moving forward for integrating with Mixer. Initial documentation on
+how to build an out-of-process adapter is provided by the
+[Out Of Process gRPC Adapter Dev Guide](https://github.com/istio/istio/wiki/Out-Of-Process-gRPC-Adapter-Dev-Guide)
+and the [gRPC Adapter Walk-through](https://github.com/istio/istio/wiki/gRPC-Adapter-Walkthrough).
+
+- **Client-Side Telemetry**. It's now possible to collect telemetry from the client of an interaction,
+in addition to the server-side telemetry.
+
+### Adapters
+
+- **SignalFX**. There is a new [`signalfx`](/docs/reference/config/policy-and-telemetry/adapters/signalfx/) adapter.
+
+- **Stackdriver**. The [`stackdriver`](/docs/reference/config/policy-and-telemetry/adapters/stackdriver/) adapter has been substantially enhanced in this
+release to add new features and improve performance.
+
 ## Security
 
-## Telemetry
+- **Authorization**. We've reimplemented our [authorization functionality](/docs/concepts/security/#authorization). Simple RPC-level
```

Comment:
Can we remove the word "simple"? "RPC-level" already explains what it does. Functionality wise, it is close to the RpcSecurityPolicy we have in Google.